demo/psa/prov-verif-e2e/README.md

```diff
@@ -70,29 +126,14 @@ Then start the VTS service:
 ```
 VTS Service starts all the supported plugins (scheme-psa-iot, scheme-tcg-dice, scheme-tpm-enacttrust for now)
 
-In the third shell
-
-```shell
-git clone https://github.com/veraison/docs
-```
+Go back to shell-1, used for Creation of PSA Endorsements
 
-Move to the docs/demo/psa/prov-verif-e2e directory:
+Ensure that you are under docs/demo/psa/prov-verif-e2e directory, otherwise, execute the command:
```

Comment:
```suggestion
Ensure that you are under docs/demo/psa/prov-verif-e2e directory:
```

(there is no issue with executing the following command even if the user already is in the correct location, so might as well simply the instructions by removing a decision branch).